app/routes/vet/milk-test-bvd.js

```diff
@@ -0,0 +1,47 @@
+const Joi = require('joi')
+const { vetVisitData: { milkTestBvdResult } } = require('../../session/keys')
+const session = require('../../session')
+const getYesNoInvestigateRadios = require('../helpers/yes-no-investigate-radios')
+
+const id = 'milkTestBvdResult'
+const errorText = 'Select one option'
```

Comment:
Same as above, the error message should be phrased as `Select yes if BVD was found in the herd`.